xbmc/music/MusicDatabase.cpp

```diff
@@ -10823,8 +10869,7 @@ void CMusicDatabase::SetPropertiesFromAlbum(CFileItem& item, const CAlbum& album
   if (album.iVotes > 0)
     item.SetProperty("album_votes", album.iVotes);
 
-  if (album.bBoxedSet)
-    item.SetProperty("album_bBoxedSet", true); //! @todo Set listItem the same way as totalDiscs ??
+  item.SetProperty("album_isboxset", album.bBoxedSet); //! @todo Set listItem the same way as totalDiscs ??
   item.SetProperty("album_totaldiscs", album.iTotalDiscs);
   item.GetMusicInfoTag()->SetTotalDiscs(album.iTotalDiscs); // sets for ListItem.TotalDiscs
```

Comment:
Remove, just isn't necessary MusicInfoTag gets populated from album by `CMusicInfoTag::SetAlbum` including totaldiscs

Reply:
> I haven't removed the line you said was unnecessary in CMusicDatabase as it makes me wonder why the other stuff is set there?

Total discs is held inside CMusicInfoTag and set by the setter, it is only the album_XXX properties that need setting here. They contain album data when the item is a song in the item properties because the song data is in the CMusicInfoTag.

You really can remove it.